interface/src/stake_history.rs

```diff
@@ -1,19 +1,19 @@
 //! A type to hold data for the [`StakeHistory` sysvar][sv].
 //!
 //! [sv]: https://docs.solanalabs.com/runtime/sysvars#stakehistory
-//!
-//! The sysvar ID is declared in [`sysvar::stake_history`].
-//!
-//! [`sysvar::stake_history`]: crate::sysvar::stake_history
```

Comment:
You can do this in follow-up work, but how about bringing in `solana-sysvar-id` crate and implementing it for stake history in this crate?

Reply:
Yeah, I will bring the stake history sysvar on a separate PR. 👍